src/sagemaker/remote_function/client.py

```diff
@@ -756,13 +1004,15 @@ def wait(
         self,
         timeout: int = None,
     ) -> None:
-        """Wait for the underlying sagemaker job to complete.
+        """Wait for the underlying SageMaker job to complete.
+
+        This method waits for the SageMaker job created as a result of the remote function execution
+        to complete for up to the timeout value (if specified). If timeout is ``None``, this method
+        will block until the job is completed.
 
-        This method blocks on the sagemaker job completing for up to the timeout value (if
-        specified). If timeout is ``None``, this method will block until the job is completed.
         Args:
-            timeout (int): Timeout in seconds to wait until the job is completed. ``None`` by
-                default.
+            timeout (int): Timeout in seconds to wait for until the job is to completed before it is
+              stopped. Defaults to ``None``.
```

Comment:
`job is to completed` sounds grammatically incorrect

Reply:
corrected.